test/widgets/goals.ts

```diff
@@ -58,7 +57,6 @@ function testGoal(goalType: string) {
       font: 'Roboto',
     };
 
-    await formMonkey.fill(testSet1);
```

Comment:
if we remove this doesn't it kinda defeat the purpose of the test?

Reply:
I fill the form in the line below. This line is not needed